<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>692</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>JetBrains Mono</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>K6GTE AuGratin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(45,45,45);
color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,3">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_band">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>40</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>80</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>160</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="time">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_mode">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>-FT*</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CW</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SSB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FT8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>FT4</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>JetBrains Mono</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Your Call:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mycall_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Your Grid:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mygrid_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
          <item>
           <widget class="QLineEdit" name="activator_call">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Activator Call</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="activator_name">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>Activator Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
          <item>
           <widget class="QLineEdit" name="park_designator">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>XX-####</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="park_name">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>Park Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
          <item>
           <widget class="QLineEdit" name="park_section">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>Section</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="park_state">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>State</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="park_grid">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Grid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="park_distance">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Km</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="park_direction">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Deg</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="time_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>HHMMSS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="date_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>YYYYMMDD</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,1,1,1,1,1,1">
          <item>
           <widget class="QLineEdit" name="band_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>000M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="freq_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>000000000</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="mode_field">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>RST TX:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rst_sent">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="placeholderText">
             <string>599</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="text">
             <string>RST RX:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rst_recieved">
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>599</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPlainTextEdit" name="comments">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>JetBrains Mono</family>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="placeholderText">
             <string>Comments</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="log_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>70</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::ClickFocus</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color:rgb(0,128,0);
color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Log It</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QWebEngineView" name="mapview" native="true">
            <property name="url" stdset="0">
             <url>
              <string>about:blank</string>
             </url>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
